GooglePay/src/main/java/com/braintreepayments/api/GooglePayClient.java

```diff
@@ -154,21 +154,16 @@ public void isReadyToPay(@NonNull final FragmentActivity activity,
      */
     public void getTokenizationParameters(
             @NonNull final GooglePayGetTokenizationParametersCallback callback) {
-        braintreeClient.getAuthorization((authorization, error) -> {
-            if (authorization != null) {
-                braintreeClient.getConfiguration((configuration, e) -> {
-                    if (configuration == null) {
-                        callback.onResult(null, null);
-                        return;
-                    }
-                    callback.onResult(
-                            getTokenizationParameters(configuration, authorization),
-                            getAllowedCardNetworks(configuration));
-                });
-            } else {
+        braintreeClient.getConfiguration((configuration, e) -> {
+            if (configuration == null) {
                 callback.onResult(null, null);
```

Comment:
Unrelated, but this is like the only callback with two success parameters and no error parameter. We should be able to fix that for v5 since we can make breaking changes.